docs/content/latest/api/ycql/ddl_create_table.md

```diff
@@ -244,9 +253,9 @@ cqlsh:example> SELECT * FROM sensor_data;
 
 ```
 
-### Create CDC table specifying number of tablets
+### Create a CDC table specifying the number of tablets
 
-For two data center (2DC) deployments that require the identical number of tablets on both clusters, you can use the `CREATE TABLE` statement with the `WITH` clause to specify the number of tablets.
+For deployments that use change data capture (CDC) and require the identical number of tablets on two YugabyteDB clusters, you can use the `CREATE TABLE` statement with the `WITH` clause to specify the number of tablets.
```

Comment:
```suggestion
You can use the `CREATE TABLE` statement with the `WITH tablets = <num>` clause to specify the number of tablets for a table. This is useful to scale the table up / down based on requirements. For example, for smaller, static tables, it may be wasteful to have a large number of shards (tablets). In that case, you can use this to reduce the number of tablets created for the table. Similarly, for a very large table, you can use this statement to pre-split the table into a large number of shards to get improved performance.

Note that yugabyteDB, by default, pre-splits a table in `yb_num_shards_per_tserver * num_of_tserver` shards. This clause can be used to override that setting on per-table basis.
```